### ISSUE TITLE
Timeout when NEO field is saved with empty value

### ISSUE THREAD

**Issue by maxstrebel** [spicywebau/craft-neo#633]
#### Description

If you have "autosaveDrafts" set to false in general.php and you create a new entry, an empty (redactor) field will make your backend unresponsive on save.

#### Steps to reproduce
1. Create a text/redactor field
2. Create a NEO field
3. Add text field to NEO
4. Set `"autosaveDrafts" => false` in `general.php`
5. Create a new entry
6. Add a NEO block, but leave the (required) text field empty
7. Save and wait forever

#### Other information
- Neo version: 3.2.3
- Craft version: 4.2.0.2
- (if multi-site) What is the affected Neo field's propagation method: Save to sites in same site group
- Is eager-loading used?

**Comment by ttempleton** [comment]
I'm unable to reproduce this. Could you please confirm whether it also happens with Matrix fields?

**Comment by maxstrebel** [comment]
Happens on my current install and on a completely new Craft 4 Instance with test fields.

Matrix will work.
NEO will throw a Bad Gateway error.

<img width="1491" alt="image" src="https://user-images.githubusercontent.com/910594/182174597-c56f508b-79a0-4966-8184-2f4a2894fcc0.png">

<img width="1491" alt="image" src="https://user-images.githubusercontent.com/910594/182174643-547cbf1e-4dee-4e52-9cff-692db2517b4b.png">

<img width="1491" alt="image" src="https://user-images.githubusercontent.com/910594/182174666-92df732e-fb48-4674-9d74-224f67fd7eb6.png">

Log will show this, though:

```
2022-08-01 14:39:10 [web.INFO] [craft\services\Elements::_saveElementInternal] Element not saved due to validation error: Array
(
    [title] => Array
        (
            [0] => Title cannot be blank.
        )

    [neoField[1].redactorText] => Array
        (
            [0] => Redactor Text cannot be blank.
        )

)
 {"memory":23003432} 
```

**Comment by ttempleton** [comment]
Ah, it's a child block. Yes, I'm getting an error in that case, although I'm getting the 'Unable to nest Neo fields' error which is clearly wrong.

**Comment by ttempleton** [comment]
That should be fixed now for the next release. In the meantime, you can get the fix by setting your composer.json file's `spicyweb/craft-neo` requirement to `3.x-dev as 3.2.4` and running `composer update`.

**Comment by maxstrebel** [comment]
Thanks a lot, Thomas! We'll start testing today.

**Comment by maxstrebel** [comment]
Thanks for fixing this so fast! Works now. 

But unfortunately the blocks do not validate. But Matrix has the same problem. Just wanted to let you know. https://github.com/craftcms/cms/issues/11731

**Comment by ttempleton** [comment]
Thanks for letting me know about that, I'll subscribe to that issue in case any changes are required on Neo's end to fix it.

Thanks also for confirming that this issue is fixed - I've just released Neo 3.2.5 including that fix.
